api/src/Page/Shipment.php

```diff
@@ -2210,7 +2210,10 @@ function _add_shipment() {
                 $tools_enclosed = $this->arg('ENCLOSEDTOOLS') ? "Yes" : "No";
             }
 
-            $dynamic = $this->has_arg("DYNAMIC") ? $this->arg("DYNAMIC") : null;
+            $dynamic = null;
+            if ($this->has_arg('DYNAMIC')){
+                $dynamic = $this->arg("DYNAMIC") ? "Yes" : "No";
```

Comment:
It seems that shipments can't be created with 'Yes'/'No', but *can* be updated with 'Yes'/'No' - which is strange. This is not an issue when going through the front-end, but does make the 'API' less sensible. Since this problem also applies to `MICROFOCUSBEAM`, `LASTMINUTEBEAMTIME` etc., perhaps fixing this should be a seperate ticket?

Reply:
Since its not relevant to updating we can leave this as is.